daffodil-runtime1/src/main/scala/org/apache/daffodil/runtime1/processors/ProcessorStateBases.scala

```diff
@@ -404,8 +404,8 @@ abstract class ParseOrUnparseState protected (
    */
   final def variableMapForDebugger = variableMap
 
-  final protected var _processorStatus: ProcessorResult = Success
-  final protected var _validationStatus: Boolean = true
+  final var _processorStatus: ProcessorResult = Success
+  final var _validationStatus: Boolean = true
```

Comment:
This is surprising, why can't these be protected?

Reply:
We get this error

```
PState.scala:666:9 
[error] 666 |      ps._processorStatus = this.processorStatus
[error]     |      ^^^^^^^^^^^^^^^^^^^
[error]     |variable _processorStatus in class ParseOrUnparseState cannot be accessed as a member of (ps : org.apache.daffodil.runtime1.processors.parsers.PState) from class Mark.
[error]     |  protected variable _processorStatus can only be accessed from class ParseOrUnparseState in package org.apache.daffodil.runtime1.processors or one of its subclasses.
```

Scala 3 tightened things up so protected members cannot be accessed from instances of subsclasses. Only instances of the class or directly from the subclass... I ran into this issue in another PR